microsoft/testsuites/core/temp_vm_hot_resize.ps1

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+<#
+.Description
+    It deploys the VM, verifies core number/memory size,
+    then resize the VM, verifies new core number/memory size.
+#>
+
+param([object] $AllVMData)
+
+function Main {
+    $testResult = ""
+
+    try {
+        $testedVMSizes = @()   # The VM sizes have been tested
+        $resizeVMSizeFailures = @()
+        $testedVMSizes += $AllVMData.InstanceSize
+        $previousVMSize = $AllVMData.InstanceSize
+
+        $loadBalanceName = (Get-AzLoadBalancer -ResourceGroupName $AllVMData.ResourceGroupName).Name
+        $VirtualMachine = Get-AzVM -ResourceGroupName $AllVMData.ResourceGroupName -Name $AllVMData.RoleName
```

Comment:
@squirrelsc is there a tool that deals with these az commands? If i'll need to implement, how can I execute powershell commands from within LISAv3? So far I have only done linux bash commands.

Reply:
In v3, we use Azure Python SDK. It has almost the same functionality like az commands. There are a lot of examples in `sut_orchestrator/azure/platform_.py`.